jwst/nsclean/nsclean.py

```diff
@@ -88,7 +92,7 @@ def mask_slits(input_model, mask):
         output mask with additional flags for slit pixels
     """
 
-    from jwst.extract_2d.nirspec import offset_wcs
+    from jwst.extract_2d.nirspec import _toindex
```

Comment:
I think this should be 
`from gwcs.utils import _toindex`

Reply:
Fixed, thank you.